docs/guides/qiskit-code-assistant-vscode.mdx

```diff
@@ -8,7 +8,7 @@ description: How to install, use, and configure Qiskit Code Assistant in Visual
 Learn how to install, use, configure and uninstall the official Qiskit Code Assistant extension in Visual Studio Code (VS Code).
 
 <Admonition type="note" title="Notes">
-    - This is an experimental feature available only to IBM Quantum&reg; Premium Plan users with [IBM Quantum Platform Classic](https://quantum.ibm.com/) accounts.
```

Comment:
This used the word Classic. It'd be helpful to search for "classic" as part of this PR

Reply:
Thanks! I left this one intentionally bc it will be changing with the new Qiskit Code Assistant PRs.